source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -128,6 +139,21 @@ content: |
             const base64DataKeyId = key.toString('base64');
             console.log("DataKeyId [base64]: ", base64DataKeyId);
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           from bson.codec_options import CodecOptions
+           from bson.binary import (STANDARD)
+           OPTS = CodecOptions(uuid_representation=STANDARD)
+           
+           connection_string = "mongodb://localhost:27017";
+           key_vault_namespace = "encryption.__keyVault";
+           fle_opts = AutoEncryptionOpts(kms_providers, key_vault_namespace, mongocryptd_bypass_spawn = True)
+           client = MongoClient(connection_string, auto_encryption_opts = fle_opts)  
+           client_encryption = pymongo.encryption.ClientEncryption(kms_providers, key_vault_namespace, client, OPTS)
+           data_key = client_encryption.create_data_key("local")
```

Comment:
is this supposed to be "local" when using aws?

Reply:
Fixed:
* Switched "local" to "aws"